HUDOC.js

```diff
@@ -0,0 +1,603 @@
v c.</span><span class='s1A844BC0'>&#xa0;</span><span class='s1A844BC0'>Chypre et Russie</span><span class='sFBBFEE58'>, 25965/04, 7</span><span class='sFBBFEE58'>&#xa0;</span><span class='sFBBFEE58'>janvier 2010, </span><a class='s37BD6B97' href=\"http://hudoc.echr.coe.int/fre?i=002-1143\"><span class='s3CA6E50F'>Note d’information 126</span></a><span class='sFBBFEE58'>&#xa0;</span><span class='sFBBFEE58'>; </span><span class='s1A844BC0'>Trabelsi c.</span><span class='s1A844BC0'>&#xa0;</span><span class='s1A844BC0'>Belgique</span><span class='sFBBFEE58'>, 140/10, 4</span><span class='sFBBFEE58'>&#xa0;</span><span class='sFBBFEE58'>septembre 2014, </span><a class='s37BD6B97' href=\"http://hudoc.echr.coe.int/fre?i=002-10080\"><span class='s3CA6E50F'>Note d’information 177</span></a><span class='sFBBFEE58'>&#xa0;</span><span class='sFBBFEE58'>; </span><span class='s1A844BC0'>Avotiņš c.</span><span class='s1A844BC0'>&#xa0;</span><span class='s1A844BC0'>Lettonie</span><span class='sFBBFEE58'> [GC], 17502/07, 23</span><span class='sFBBFEE58'>&#xa0;</span><span class='sFBBFEE58'>mai 2016, </span><a class='s37BD6B97' href=\"http://hudoc.echr.coe.int/fre?i=002-11062\"><span class='s3CA6E50F'>Note d’information 196</span></a><span class='sFBBFEE58'>&#xa0;</span><span class='sFBBFEE58'>; </span><span class='s1A844BC0'>Pirozzi c.</span><span class='s1A844BC0'>&#xa0;</span><span class='s1A844BC0'>Belgique</span><span class='sFBBFEE58'>, </span><a class='s37BD6B97' href=\"http://hudoc.echr.coe.int/fre?i=001-182231\"><span class='s3CA6E50F'>21055/11</span></a><span class='sFBBFEE58'>, 17</span><span class='sFBBFEE58'>&#xa0;</span><span class='sFBBFEE58'>avril 2018</span><span class='sFBBFEE58'>&#xa0;</span><span class='sFBBFEE58'>; et </span><span class='s1A844BC0'>Güzelyurtlu et autres c.</span><span class='s1A844BC0'>&#xa0;</span><span class='s1A844BC0'>Chypre et Turquie </span><span class='sFBBFEE58'>[GC], 36925/07, 29</span><span class='sFBBFEE58'>&#xa0;</span><span class='sFBBFEE58'>janvier 2019, </span><a class='s37BD6B97' href=\"http://hudoc.echr.coe.int/fre?i=002-12316\"><span class='s3CA6E50F'>Note d’information 225</span></a><span class='sFBBFEE58'>&#xa0;</span><span class='sFBBFEE58'>; ainsi que la fiche thématique </span><a class='s37BD6B97' href=\"https://www.echr.coe.int/Documents/FS_European_Union_FRA.pdf\"><span class='s3CA6E50F'>Jurisprudence relative à l’Union européenne</span></a><span class='sFBBFEE58'>)</span></p><p class='sBE3830CA'><span class='sFBBFEE58'>&#xa0;</span></p><p class='s6090DB7C'><span class='sFBBFEE58'>© Conseil de l’Europe/Cour européenne des droits de l’</span><span class='sFBBFEE58'>homme</span><br /><span class='sFBBFEE58'>Rédigé par le greffe, ce résumé ne lie pas la Cour.</span></p><p class='s6090DB7C'><span class='sFBBFEE58'>Cliquez ici pour accéder aux </span><a class='s37BD6B97' href=\"http://www.echr.coe.int/NoteInformation/fr\"><span class='s3CA6E50F'>Notes d'information sur la jurisprudence</span></a></p></div>"
```

Comment:
Could we clean up the HTML a bit? A lot of characters that don't do anything. I'm thinking remove all span tags and all class attributes? Any reason not to?